Bottom: d86d0baadd9e76015234d9fe97477781a201c794
Top:    4a7dab2b01b7aaaed4b23c5e3e3713745e6e7124
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-11-20 00:57:03 +0000

Refresh of play-with-glsl-shader-for-rend

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 0a45934..3448565 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -48,7 +48,6 @@ RCSID ("$Id: $");
 triangle_buffer buffer;
 float global_depth = 0;
 hidgl_shader *circular_program = NULL;
-hidgl_shader *resistor_program = NULL;
 
 static bool in_context = false;
